root/release/add_cover_art.tt

```diff
@@ -18,7 +18,9 @@
     }).fail(function () { $(".caa-warning").parent().toggle(); });
   </script>
 
-  <form id="add-cover-art" class="cover-art" action="[% c.req.uri %]" method="post" >
+  [%# Remove query params used to seed fields to avoid a "field must contain a single
+      value" error due to submitting duplicate parameters via the body later. %]
+  <form id="add-cover-art" class="cover-art" action="[% c.req.uri.replace('\?.*', '') %]" method="post" >
```

Comment:
I'm guessing we can just use `c.req.uri.path` here to omit the query parameters, but I haven't tested it.
```suggestion
  <form id="add-cover-art" class="cover-art" action="[% c.req.uri.path %]" method="post">
```

Reply:
Thanks, I was hoping that something like that was available. I can't test this right now either, but I think it'll be exercised by the Selenium test.